src/topology/continuous_function/compact.lean

```diff
@@ -188,6 +188,28 @@ instance : normed_ring C(α,R) :=
 { norm_mul := λ f g, norm_mul_le (mk_of_compact f) (mk_of_compact g),
   ..(infer_instance : normed_group C(α,R)) }
 
+lemma continuous_const [nonempty α] : continuous (continuous_map.const : R → C(α, R)) :=
```

Comment:
This will follow from https://github.com/leanprover-community/mathlib/blob/9fd7a02d03ba2ea9786eea4bf547b2c51e56ed6a/src/topology/compact_open.lean#L302 once we merge #11229 

I suggest we wait for that.